scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -83,13 +84,15 @@ def get_sem_rotation(ccd):
 
 def main(args):
     """
-    Handles the command line arguments
-    args is the list of arguments passed
-    return (int): value to return to the OS as program exit code
+    Handles the command line arguments.
+
+    args: The list of arguments passed.
+    return:
+        (int) value to return to the OS as program exit code.
     """
     parser = argparse.ArgumentParser()
     parser.add_argument('--debug', action="store_true", default=False)
-    parser.add_argument("--role", dest="role", metavar="<component>",
+    parser.add_argument("--role", dest="role", default="diagnostic-ccd", metavar="<component>",
```

Comment:
Isn't it `Role of the camera to connect to via the Odemis back-end`

Reply:
that is correct, I'll update it